uber/models.py

```diff
@@ -1215,7 +1215,11 @@ def total_cost(self):
 
     @property
     def amount_unpaid(self):
-        return max(0, self.total_cost - self.amount_paid)
+        if self.paid == c.PAID_BY_GROUP:
```

Comment:
this is a little OMHBITY, man these cost properties are so tough to keep track of mentally when reading the code at a glance.